src/routes/firstQuestions.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const firstQuestionRouter = express.Router;
+const registerQuestions = require('../components/questions/firstQuestionModel');
+
+//post questions on register
+firstQuestionRouter.post('/register-question', (req, res) => {
```

Comment:
Are we going to have a secondQuestionRouter? 

I think it's a good idea to have only one questions model/controller/router that takes care of all the questions.

Same goes for answers.